src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir-Heavy, serif;
+}
+
+.body {
+  margin: 0 8px;
+}
+
+form[data-qa="big"] {
```

Comment:
use class selectors for styling  everywhere

Reply:
change to only classes